onchain/src/social_account.cairo

```diff
@@ -78,9 +76,9 @@ mod tests {
         let (_, event) = spy.events.at(0);
         assert(event.keys.at(0) == @selector!("AccountCreated"), 'Wrong event name');
 
-        let key_felt252 = *event.keys.at(1);
-        let key: u256 = key_felt252.into();
-        assert(key == public_key, 'Wrong Public Key');
+        let event_key: u256 = (*event.keys.at(1)).into();
```

Comment:
Do you really need the type declaration here?

Reply:
Yes, in order to type cast it as u256

Reply:
but if this is clean i can change it 

  ` assert(public_key == (*event.keys.at(1)).into(), 'Wrong Public Key');`

Reply:
this work         
```
let event_key = (*event.keys.at(1)).into();

assert(event_key == public_key, 'Wrong Public Key');
```